lib/services/ngsi/entities-NGSI-v2.js

```diff
@@ -541,6 +541,8 @@ function sendUpdateValueNgsi2(entityName, attributes, typeInformation, token, ca
         } else if (typeInformation.explicitAttrs && typeof typeInformation.explicitAttrs === 'boolean') {
```

Comment:
This PR needs to modify CNR? Or it is already covered by an existing entry?

Reply:
No CNR is expected to be modified with this PR

Reply:
NTC